openlibrary/macros/UserMetadata.html

```diff
@@ -0,0 +1,31 @@
+$def with (work, edition)
+
+$ username = ctx.user and ctx.user.key.split('/')[-1]
+$ has_edition = edition is not None
+
+$ i18n_strings = {
+  $ "close_text": _("Close"),
+  $ "submit_text": _("Submit")
+  $ }
+
+$ context = {
+  $ "username": username,
+  $ "work": work.key
+  $ }
+
+$if has_edition:
+  $ context["edition"] = edition.key
+
+<div class="cta-section" style="text-align:center; margin-top:7px;">
```

Comment:
it would be great for these inlines to go into our styles, but fine for this mvp